### DISCUSSION TITLE
laravel 8 jetstram doesn't work with oci adw and/or oci atp

### DISCUSSION THREAD

**Discussion by cadentic** [laravel/framework#38918] [Q&A]
hi

laravel 8 and/or jetstream doesn't work with oci adw and/or oci atp?
yajra/laravel-oci8 was not enough to deal with it.
is there any terraform available for this ? 

oracle autonomous linux
oracle autonomous database 19c
oic client 21c

thx
sayantan

**Comment by crynobone** [comment]
Hi there,

Laravel Framework only officially supports SQLite, MySQL, Postgres and MSSQL at the moment.

**Reply by cadentic** [reply]
this was not actually fully concerned with Laravel x , it was also related to Yajra/laravel-oci8. it doesn't matter what Laravel supports or not if you have a package that offers this connectivity it would work with Laravel x. so my concern was also with Yajra/laravel-oci8. 
hope it will explain my situation

**Reply by crynobone** [reply]
We don't official support or maintain `Yajra/laravel-oci8`. You need to report the issue to the actual project maintainers to solve this issue.
